docs/how-to-setup-mobile-app-locally.md

```diff
@@ -0,0 +1,140 @@
+
+Run these commands on your local machine:
+
+1. Open a Terminal / Command Prompt / Shell in your projects directory
+   i.e. `/yourprojectsdirectory/`
+2. Clone your fork of freeCodeCamp, replacing YOUR_USER_NAME with your GitHub Username
+
+```
+ git clone https://github.com/YOUR_USER_NAME/mobile.git
+```
+
+Next change the directory:
+
+```
+cd mobile
+```
+
+Next set the upstream branch to:
+
+```
+git remote add upstream https://github.com/freeCodeCamp/mobile.git
+```
+
+Make sure everything looks correct:
+
+```
+git remote -v
+```
+
+The output should look somehting like below (replacing `YOUR_USER_NAME` with your Github username):
+
+```
+origin    https://github.com/YOUR_USER_NAME/mobile.git (fetch)
+origin    https://github.com/YOUR_USER_NAME/mobile.git (push)
+upstream    https://github.com/freeCodeCamp/mobile.git (fetch)
+upstream    https://github.com/freeCodeCamp/mobile.git (push)
+```
+
+## Setting up IDE and Flutter
+
+1. first you will have to install Flutter on your local machine - https://docs.flutter.dev/get-started/install, we currently make use of Flutter `2.8.x`. Follow the instructions carefully.
+2. Next set up (or install) an IDE to make developing the app a faster experience. - We mainly make use of Visual Studio Code but installing Android Studio is also recommended to have easy access to the AVD-manager. (The manager where you access your installed emulators)
+3. Make sure to install the recommended Flutter plugin if you use VsCode. To do this:
+   - Start VsCode
+   - Invoke `View > Command Palette`
+   - Type `install`, and select `Extensions: Install Extensions`
+4. Now validate your setup with the Flutter doctor.
+   To do this: - `Invoke View > Command Palette` - Type `doctor`, and select the `Flutter: Run Flutter Doctor`
+
+It should look something like this:
+
+```
+Doctor summary (to see all details, run flutter doctor -v):
+[√] Flutter (Channel stable, 2.5.3, on Microsoft Windows [Version 10.0.19042.1415], locale nl-NL)
+[√] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
+[√] Chrome - develop for the web
+[√] Android Studio (version 4.1)
+[√] VS Code, 64-bit edition (version 1.60.1)
+[√] Connected device (2 available)
+
+• No issues found!
+```
+
+if not go to the trouble shoot section.
+
+## Installing emulators
+
+When installing an emulator from `Android studio` we recommend using the `Pixel 3a XL` and `Nexus One` for smaller screens. If you use any other IDE, we recommend installing the emulators from `Android Studio`.
+
+The installing of a new emulator is pretty straightforward. Firstly you will have to open up `Android Studio`. Make sure you have cloned the repository from the `freeCodeCamp` organization. Open up the project in `Android studio`. Now search for the 3rd icon in the top-right corner. It looks like a mobile phone with a green Android next to it. When hovering over it, it should say `AVD manager.
```

Comment:
This is a bit specific. I say that as a problem, because what if Android decide to make it the `4th` icon in the bottom-left corner?

```suggestion
Open `Android Studio`. Ensure you have cloned the repository from the `freeCodeCamp` organization. Open up the project in `Android studio`. Now search for the 3rd icon in the top-right corner. It looks like a mobile phone with a green Android next to it. When hovering over it, it should say `AVD manager`.
```